11_packages.qmd

```diff
@@ -0,0 +1,111 @@
+---
+title: "11_packages"
+editor: visual
+---
```

Comment:
This is a much sparser YAML header than the other chapters.

Reply:
Yes! This is true. I commented on this in the Slack message, but the quarto document is **not** ready for primetime and is not rendered in the book. The header will 100% need to be updated.